nest-forms-backend/src/ginis/ginis.service.ts

```diff
@@ -549,11 +552,19 @@ export default class GinisService {
       await this.prismaService.forms.update({
         where: { id: form.id },
         data: {
-          state: FormState.PROCESSING,
+          state: formDefinition.sharepointData
+            ? FormState.SENDING_TO_SHAREPOINT
+            : FormState.PROCESSING,
           ginisState: GinisState.FINISHED,
           error: FormError.NONE,
         },
       })
+
+      if (formDefinition.sharepointData) {
+        await this.sendToSharepoint(form.id)
+      }
+
+      // TODO do we want to send the mail now? Or after sharepoint delivery
```

Comment:
Question: Do we want to send the email confirming form delivery now, or after delivery to sharepoint?

Reply:
What is the email that we should send? It's the first time I hear about this.

Reply:
The sending happens under this comment

Reply:
Ah, I think it's OK here, but I think you can make better educated guess.

Reply:
Yes let's leave it here, if it is in Ginis, we can say that it is successfuly delivered.